app/src/androidTest/java/com/hedvig/app/feature/referrals/ReferralScreen.kt

```diff
@@ -1,9 +1,58 @@
 package com.hedvig.app.feature.referrals
 
-import com.agoda.kakao.common.views.KView
+import android.content.Intent
+import android.view.View
+import com.agoda.kakao.image.KImageView
+import com.agoda.kakao.intent.KIntent
+import com.agoda.kakao.recycler.KRecyclerItem
+import com.agoda.kakao.recycler.KRecyclerView
 import com.agoda.kakao.screen.Screen
+import com.agoda.kakao.text.KButton
+import com.agoda.kakao.text.KTextView
 import com.hedvig.app.R
+import org.hamcrest.Matcher
 
 class ReferralScreen : Screen<ReferralScreen>() {
```

Comment:
Should this be named `ReferralScreenTest`? it just thought me of with using all KImageView and KTextView and never doing anything with the values

Reply:
`ReferralScreen` is simply a representation of the elements that are present on this particular screen, a screen object model, if you will. It does not test anything on its own, it is only used in other tests